### PR TITLE
Added script and support files to install ipython and jupyter notebooks

### PR DESCRIPTION
**Pull request by epifanio**
This PR adds:

- updated version of IPython
- Jupyter notebook
- additional kernels (R, python3, bash, octave)
- scientific packages for python3

tested on both arch (i386, amd64)
p.s.
I'm having trouble in understanding git ...

### CODE REVIEW

File: bin/install_ipython.sh [hunk 1]
```diff
@@ -92,6 +86,225 @@ if [ ! -d "/etc/skel/.ipython/nbextensions" ] ; then
...
+
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-rasterio_0.24.1_$ARCH.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-netcdf4_1.1.9_$ARCH.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-descartes_1.0.1_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-geojson_1.2.1_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-geopandas_0.1.1_all.deb"
+
+
+
+dpkg -i python2-rasterio_0.24.1_$ARCH.deb
```
Comment (kalxas) [inline bin/install_ipython.sh]: Those packages (netcdf, descartes, geojson, geopandas,rasterio) already exist in OSGeoLive ppa. Can you please use those instead?

**Comment by kalxas** [comment]
@epifanio which of the 2 branches is the one you are working on? GSoC or GSoC2015?

File: bin/install_ipython.sh (hunk 1 above)
Comment (epifanio) [inline bin/install_ipython.sh]: I'll add the ppa and replace those packages

**Comment by epifanio** [comment]
I'm working on :  GSoC

File: bin/install_ipython.sh (hunk 1 above)
Comment (epifanio) [inline bin/install_ipython.sh]: here https://launchpad.net/~johanvdw/+archive/ubuntu/osgeolive  i found 

```fiona, descartes, geojson, rasterio, shapely```

I didn't found : 

```netcdf4-python, geopandas```

Comment (kalxas) [inline bin/install_ipython.sh]: The ppa is already included, no need to add anything.

Comment (epifanio) [inline bin/install_ipython.sh]: Not on a fresh lubuntu for a cleqn test.

Comment (kalxas) [inline bin/install_ipython.sh]: https://launchpad.net/~osgeolive/+archive/ubuntu/nightly/+packages

Comment (epifanio) [inline bin/install_ipython.sh]: Thanks!  I'll use it as guideline to build the.debs.  For now the py3 packages are no longer installed by the script.

**Comment by kalxas** [comment]
@epifanio please see this iso (built from your branch), the jupiter launcher does not seem to work.
http://aiolos.survey.ntua.gr/gisvm/dev/osgeo-live-nightly-build115-amd64-2741438.iso

**Comment by epifanio** [comment]
@kalxas i downloaded the iso and actually none of the packages get installed. I mounted the iso as a VM (without installing it) and i run the ```install_ipython.sh``` from ```/home/user/gisvm/bin/``` after executing the script everything was installed correctly. I don't know what went wrong during the iso build. perhaps the sequence of installation scripts can brake some dependencies ? What's the exact sequence of how the installation scripts are run?

**Comment by kalxas** [comment]
https://github.com/OSGeo/OSGeoLive/blob/master/bin/inchroot.sh#L120

**Comment by kalxas** [comment]
@epifanio http://aiolos.survey.ntua.gr/gisvm/dev/osgeo-live-nightly-build115-amd64-2741438-log.tar.gz will tell you the story...

**Comment by epifanio** [comment]
I checked the log, at line ```5026``` in chroot-build.log : 

 https://gist.github.com/4771c564345d0cac26bf

the script was called without ```arch``` argument

**Comment by kalxas** [comment]
New test build 

http://aiolos.survey.ntua.gr/gisvm/dev/osgeo-live-nightly-build105-amd64-537560e.iso

**Comment by kalxas** [comment]
From a first look, I don't see the reason to have both IPython launchers and Jupiter ones, both open up Jupiter, right?

**Comment by epifanio** [comment]
@kalxas ,  that's correct they both open the same notebook (jupyter)  
the ipython launchers can be removed (I update the installer)

**Comment by kalxas** [comment]
Since development moved to another branch, I am closing this for now

**Pull request closed without merging**